components/automate-ui/src/app/modules/infra-proxy/create-data-bag-modal/create-data-bag-modal.component.html

```diff
@@ -5,12 +5,12 @@ <h2 slot="title">Create Data Bag</h2>
       <div class="margin">
         <chef-form-field>
           <label>
-            <span class="label">Name</span>
+            <span class="label">Name <span aria-hidden="true">*</span></span>
```

Comment:
Author note: accessibility -- every required field needs an asterisk designated it as required.